src/animation/VisualElementAnimationControls.ts

```diff
@@ -552,48 +551,55 @@ export class VisualElementAnimationControls<
     }
 
     private animateVariant(variantLabel: string, opts?: AnimationOptions) {
-        let when: Orchestration["when"] = false
-        let delayChildren = 0
-        let staggerChildren = 0
-        let staggerDirection = 1
```

Comment:
Where are these defaults now defined?

Reply:
Ah they were already double defined in animateChildren